android-project/app/build.gradle

```diff
@@ -35,36 +34,37 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
-    applicationVariants.all { variant ->
-        tasks["merge${variant.name.capitalize()}Assets"]
-            .dependsOn("externalNativeBuild${variant.name.capitalize()}")
+
+    if (buildAsApplication) {
+        applicationVariants.all { variant ->
+            tasks["merge${variant.name.capitalize()}Assets"]
+                .dependsOn("externalNativeBuild${variant.name.capitalize()}")
+        }
     }
+
     if (!project.hasProperty('EXCLUDE_NATIVE_LIBS')) {
         sourceSets.main {
             jniLibs.srcDir 'libs'
         }
         externalNativeBuild {
             ndkBuild {
-                path 'jni/Android.mk'
+                path '../../Android.mk'
```

Comment:
This change doesn't make sense to me. This is supposed to point to the application's makefile, not SDL's. That's why it's pointed to the sample in the `jni` folder.